src/scenes/ContactUsRouter/ContactUsForm.tsx

```diff
@@ -145,16 +150,23 @@ const ContactUsForm = () => {
       snackbar.toastSuccess(strings.formatString(strings.THANK_YOU_FOR_CONTACTING_SUPPORT, `${issueKey}`));
       goToContactUs();
     }
-  }, [submitSupportRequest, snackbar]);
+  }, [activeLocale, submitSupportRequest, snackbar]);
 
   const supportRequestTitle = useMemo(
     () => (supportRequestType ? getSupportRequestName(supportRequestType) : ''),
-    [supportRequestType]
+    [activeLocale, supportRequestType]
   );
 
   const supportRequestInstructions = useMemo(
     () => (supportRequestType ? getSupportRequestInstructions(supportRequestType) : ''),
-    [supportRequestType]
+    [activeLocale, supportRequestType]
+  );
+
+  // Confirming that no uploads are pending.
+  const uploadCompleted = useMemo(
+    () =>
+      allAttachments.map((attachment) => attachment.status !== 'pending').reduce((prev, curr) => prev && curr, true),
```

Comment:
This can be simplified by using `.every`: `allAttachments.every((attachment) => attachment.status !== 'pending')`

Reply:
Thanks! Much cleaner!!